closed/make/copy/Copy-java.base.gmk

```diff
@@ -36,8 +36,9 @@ TARGETS += \
 	#
 
 $(OUTPUTDIR)/vm/include/jvmti.h: $(TOPDIR)/openj9/runtime/include/jvmti.h.m4
-  $(MKDIR) -p $(@D)
+	$(MKDIR) -p $(@D)
 	m4 -D JAVA_SPEC_VERSION=$(VERSION_FEATURE) $< > $@
+	@$(ECHO) "END this PRO"
```

Comment:
This is looking good. I'm assuming you'll remove this echo command once you have verified it is all working.

Reply:
build successfully in Jenkins, link to it https://hyc-runtimes-jenkins.swg-devops.com/job/Build-JDK11-linux_x86-64_cmprssptrs/1392/